### ISSUE TITLE
OrganizationChartNode Cannot read property 'styleClass' of null

### ISSUE THREAD

**Issue by DeniseCP** [primefaces/primeng#3758]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ x ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
```
ERROR TypeError: Cannot read property 'styleClass' of null
    at Object.View_OrganizationChartNode_0._co [as updateDirectives] (OrganizationChartNode.html:4)
    at Object.debugUpdateDirectives [as updateDirectives] (core.es5.js:13058)
    at checkAndUpdateView (core.es5.js:12238)
    at callViewAction (core.es5.js:12603)
    at execComponentViewsAction (core.es5.js:12535)
    at checkAndUpdateView (core.es5.js:12244)
    at callViewAction (core.es5.js:12603)
    at execComponentViewsAction (core.es5.js:12535)
    at checkAndUpdateView (core.es5.js:12244)
    at callViewAction (core.es5.js:12603)
```

> I have a service that returns an json. Once with that data I call a recursive function to build my org chart, the org chart shows on the screen but I get this error always, and I don't know where to look for solutions, since I'm using the ngOnInit as the example showed to add data to the org chart.

```
public chart: TreeNode[];
  superGroupCounter = 0;  
  groupsArray = [];

  constructor(private recordsService: RecordsService) { }

  ngOnInit() {
    this.recordsService.getGroupsJSON()
      .subscribe(res => {
        this.data = res
        this.recursive(this.data) 
        this.chart = this.groupsArray       
      })     
  }  

  recursive (obj, root?){
    var count = 0
    var superGroup = null
    var parent = 0
    for (var k in obj)
      {
        if (typeof obj[k] == "object" && obj[k] !== null) {
          if (k==="groups") { //It's a branch
            this.superGroupCounter++
            superGroup = "Group " + this.superGroupCounter
              for(let index = 0; index < obj[k].length; index++){                         
                if(obj[k][index].hasOwnProperty("groups")){
                  root['children'].push (
                    {
                      "label" : superGroup,
                      "expanded": true,
                      "data" : obj[k][index].weight ,
                      "leaf" : false,
                      "styleClass": 'ui-person'
                    })
                  this.recursive(obj[k][index], root["children"][count])
                }else {
                  if(!root['children']) {
                    root['children'] = []
                  }
                  root['children'].push (
                    {
                      "label" : obj[k][index].name,
                      "data" : obj[k][index].name ,
                      "leaf" : true,
                      "styleClass": 'ui-person'
                    })
                  count++
                }
              }                        
            }
          } else {
            if(k === 'total'){
              this.groupsArray.push(                  
                {
                  "label" : obj[k],
                  "expanded": true,
                  "data" :  obj[k].weight ,
                  "leaf" : false,
                  "type": 'root',
                  "styleClass": 'ui-person'
                })
              root = this.groupsArray[parent]
            }            
          this.recursive(obj[k], root) 
        }                  
      }
    }
```

**Please tell us about your environment:**
Windows 10, Chrome 59+,  VSCode, Node 5.3.0

**Comment by cagataycivici** [comment]
Got your twitter message, will review tomorrow 🙂

**Comment by cagataycivici** [comment]
Unable to replicate, please provide a case at;

http://plnkr.co/edit/Qi2Nw2imLpDUF8NJinYw?p=preview

**Comment by cagataycivici** [comment]
Anyway, I've added some null checks via ngIf which I believe should fix this, try with monday's 4.2.RC1 please.

**Comment by DeniseCP** [comment]
Thanks @cagataycivici it worked! No more console errors! I owe you!
